src/components/AuthContainer.js

```diff
@@ -38,6 +39,27 @@ export default class AuthContainer extends Component {
       }, () => {
         console.warn('Error looking up user status')
       })
+
+      const [billingProjects, clusters] = await Promise.all([Rawls.listBillingProjects(), Leo.clustersList()])
+      if (billingProjects && clusters) {
```

Comment:
nit: I don't think you need this check. If either call fails, you'll get an exception, so if the code gets here you can assume success. Looks good overall.